src/mono/wasm/debugger/tests/debugger-test/debugger-get-properties-test.cs

```diff
@@ -115,135 +115,12 @@ public async Task InstanceMethodAsync()
         }
     }
 
-    public class BaseBaseClass2
```

Comment:
Did you mean to remove these? The tests are failing now.

Reply:
Oh, I did not. I overlooked this change when committing. Thanks!